learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md

```diff
@@ -95,7 +99,13 @@ Follow the steps below to write a simple program in a Ballerina package, which p
 
 3. Navigate to the created `helloworld` package directory.
 
-4. Execute the `bal build` command to build an executable of this file. You view the output below.
+4. Execute the command below to build an executable of this file. You view the output below.
+
+    ```bash
+    bal build
+    ```
+
+    You view the output below.
```

Comment:
"You view the output below" segment is repeated here.

Reply:
Fixed.